project_management_flask/flask_app/templates/group_projects_dashboard.html

```diff
@@ -4,9 +4,134 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" 
+    href="https://cdn.jsdelivr.net/npm/bootstrap@4.1.3/dist/css/bootstrap.min.css" 
+    integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" 
+    crossorigin="anonymous">
+    <link rel="stylesheet" type="text/css" href="../static/style.css" />
     <title>Group Projects Dashboard Page</title>
 </head>
 <body>
-    
+    <header>
+        <nav class="navbar navbar-expand-lg navbar navbar-dark bg-dark">
+            <div class="container-fluid">
+                <a class="navbar-brand" href="#"><h3>Group Projects</h3></a>
+                <span>
+                    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+                    <span class="navbar-toggler-icon"></span>
+                    </button>
+                    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+                    <ul class="navbar-nav me-auto mb-2 mb-lg-0">
+                        <li class="nav-item ">
+                        <a class="nav-link" href="/dashboard">Return to Home |</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="/projects/{id}/edit">Edit Project |</a>
+                        </li>
+                        <li class="nav-item">
+                            <a class="nav-link" href="/logout">Logout</a>
+                        </li>
+                    </ul>
+                </span>
+                </div>
+            </div>
+        </nav>
+    </header>
+    <main class="my-5">
+        <div class="container border border-dark bg-light">
+            <h3 class="my-2">
+                {{project.name}}
+                <br>
+                There are a total of {{tasks.count}} tasks to complete for this project. {{user.tasks.count}} of them are assigned to you.
+            </h3>
+            <br>
+            <p class="align-self-end"><a href="">Add Task</a></p>
+            <br>
+            <h4>Your Tasks:</h4>
+            {% if tasks.user.count == 0 %}
+            <table class="table text-center border border-secondary"style="background-color: #e3f2fd;">
+                <thead class="thead-dark">
+                    <tr>
+                        <th>Task</th>
+                        <th>Due Date</th>
+                        <th>Assigner</th>
+                        <th>Action</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    {% for task in tasks %}
+                    <tr>
+                        <td>{{task.name}}</td>
+                        <td>{{task.dueDate}}</td>
+                        <td>{{task.assigner}}</td>
```

Comment:
forgot about flask, but do we need task.assigner.name since it is left joined to the users t able?

Reply:
No we don't, these are just place holder so I know what to put in there once we are set with task controllers